lib/pliny/helpers/params.rb

```diff
@@ -21,10 +21,21 @@ def parse_body_params
     def load_params(data)
       # Sinatra 1.x only supports the method. Sinatra 2.x only supports the class
       if defined?(Sinatra::IndifferentHash)
-        Sinatra::IndifferentHash[data]
+        indifferent_params_v2(data)
       else
         indifferent_params(data)
       end
     end
+
+    def indifferent_params_v2(data)
+      case data
+      when Hash
+        Sinatra::IndifferentHash[data.to_hash]
```

Comment:
@stevenharman I think the `to_hash` is redundant here because we have already verified it is a `Hash`

Reply:
Doh! Good catch.